### PR TITLE
IE11 Mathjax - use classNames instead of classList

### PR DESCRIPTION
**Pull request by darrenmason**
IE11 is buggy with / does not have support for classList.

http://caniuse.com/#feat=classlist
http://stackoverflow.com/questions/8098406/code-with-classlist-does-not-work-in-ie

rendering now, though still has some alignment issues.

mathjax-workin

### CODE REVIEW

File: shared/src/helpers/mathjax.coffee [hunk 1]
```diff
@@ -30,8 +30,9 @@ typesetDocument = (windowImpl) ->
 
   windowImpl.MathJax.Hub.Queue ->
     # Queue a call to mark the found nodes as rendered so are ignored if typesetting is called repeatedly
+    # uses className += instead of classList because IE
     for node in latexNodes.concat(mathMLNodes)
-      node.classList.add(MATH_RENDERED_CLASS)
+      node.className += " #{MATH_RENDERED_CLASS}"
```
Comment (nathanstitt) [inline shared/src/helpers/mathjax.coffee]: we're operating on "real" DOM Elements here, so `className` isn't a property, we'll need to use the `class` attribute directly

**Comment by philschatz** [comment]
Should we be selecting MathML or SVG nodes? I thought we'd only be selecting the `<span data-math>` elements. If we are selecting the MathML or SVG nodes, is that a bug?

**Comment by darrenmason** [comment]
@philschatz I don't think we are...  I should restate the description :)

**Comment by philschatz** [comment]
In that case, should the fix be to not select the MathML/SVG nodes in the first place? It seems continuing to select the MathML/SVG nodes would just patch the existing problem and maybe cause other problems in the future?

File: shared/src/helpers/mathjax.coffee (hunk 1 above)
Comment (darrenmason) [inline shared/src/helpers/mathjax.coffee]: I thought we could use this? https://developer.mozilla.org/en-US/docs/Web/API/Element/className

Seems to be working

**Comment by darrenmason** [comment]
This fix should work with mathml or svg should they be selected in the future

**Comment by nathanstitt** [comment]
I'm going to merge since it's conceptually the same as using `classList` and will hopefully fix the land-standing IE bugs, since we do know that IE's `classList` support isn't the best

**Pull request merged**